### ISSUE TITLE
Flatpak app

### ISSUE THREAD

**Issue by tim77** [cjbassi/ytop#63]
Have you considered make universal package which will run on all major Linux distributions? Just few pros like:

- Guarantees to work on old and new distros.
- Your app will be available out of box in modern GUI package managers like GNOME Software and KDE Discover. This is attractive for users and help to discover your app.

---

- [Flatpak website](https://www.flatpak.org/)
- [Flatpak docs](http://docs.flatpak.org/en/latest/getting-started.html)

I'll try to push initial manifest, hope this helps a little bit and you can use it as example.

**Comment by cjbassi** [comment]
I tried using snaps with gotop and i really didn't like it, but i haven't used flatpak yet and i heard it was better. I'll give it a try and if it works out then i would be open to adding support for it.

**Comment by Seirdy** [comment]
For reference, other TUI applications available as Flatpaks include Cointop and Neovim.

**Comment by tim77** [comment]
Yep. And i really like `ytop` even with there is some limitation with showing host processes. There also thing which we should mention, that Rust software sometimes problematic to package for official distros, so it's really nice to have working flatpak version.
